packages/components/app/styles/components/dropdown.scss

```
@@ -132,20 +132,36 @@ $hds-dropdown-toggle-border-radius: 5px;
   max-width: initial;
 }
 
+.hds-dropdown-list--position-bottom-right,
 .hds-dropdown-list--position-right {
   position: absolute;
   top: calc(100% + 4px);
   right: 0;
   z-index: 2; // https://github.com/hashicorp/design-system/issues/114
```

I think you can remove these comments, are not relevant anymore
```suggestion
  z-index: 2;
```